CodeEdit/InspectorSidebar/Models/HistoryInspectorModel.swift

```diff
@@ -25,7 +25,7 @@ public final class HistoryInspectorModel: ObservableObject {
 
     /// Initialize with a GitClient
     /// - Parameter workspaceURL: the current workspace URL
-    ///
+
```

Comment:
Please remove vertical whitespace

Reply:
I removed the vertical whitespace

Reply:
line 29 still needs to be deleted

Reply:
It's not deleting/updating for some reason